api/app/signals/apps/api/v1/serializers/signal.py

```diff
@@ -332,46 +332,35 @@ class PublicSignalCreateSerializer(serializers.ModelSerializer):
     class Meta(object):
         model = Signal
         fields = (
```

Comment:
Why did you removed the `signal_id`? This is still needed, right?

Reply:
fixed that locally, was clearly wrong

Reply:
Was clearly wrong, fixed now.